doc/adr/0007-api-endpoints-setup.md

```diff
@@ -0,0 +1,84 @@
+# 7. API endpoints setup
+
+Date: 2024-03-07
+
+## Status
+
+Accepted
+
+## Context
+
+We need to support a growing list of endpoints. At the same time, we want to avoid repeated discussions about what each endpoint should look like, and reduce the time spent thinking about which information should be available where.
+
+This ADR outlines our idea of sensible defaults for how endpoints should be structured, and how they should behave. The goal is to have a shared understanding that makes API endpoint-related decisions easier, and the structure and behavior of those endpoints more predictable.
+
+## Decision
+
+Our proposal consists of a simple foundation that, while not ideal, supports everything we need. It is also easy to extend to support requirements we will very likely have in the near future.
+
+We're generally not dogmatic, but we're aiming for a REST-ful interface, meaning that we ask the API for resources, not functionality.
```

Comment:
Maybe the fact first: "We're aiming for a RESTful interface. meaning ... - but we're not dogmatic about it."